jiant/shared/model_setup.py

```diff
@@ -5,43 +5,6 @@
 from jiant.shared.model_resolution import ModelArchitectures, resolve_tokenizer_class
 
 
-def get_tokenizer(model_type, tokenizer_path):
-    """Instantiate a tokenizer for a given model type.
-
-    Args:
-        model_type (str): model shortcut name.
-        tokenizer_path (str): path to tokenizer directory.
-
-    Returns:
-        Tokenizer for the given model type.
-
-    """
-    model_arch = ModelArchitectures.from_model_type(model_type)
-    tokenizer_class = resolve_tokenizer_class(model_type)
-    if model_arch in [ModelArchitectures.BERT]:
-        if "-cased" in model_type:
-            do_lower_case = False
-        elif "-uncased" in model_type:
-            do_lower_case = True
-        else:
-            raise RuntimeError(model_type)
-    elif model_arch in [
-        ModelArchitectures.XLM,
-        ModelArchitectures.ROBERTA,
-        ModelArchitectures.XLM_ROBERTA,
-        ModelArchitectures.BART,
-        ModelArchitectures.MBART,
-        ModelArchitectures.ELECTRA,
-    ]:
-        do_lower_case = False
-    elif model_arch in [ModelArchitectures.ALBERT]:
-        do_lower_case = True
-    else:
-        raise RuntimeError(str(tokenizer_class))
-    tokenizer = tokenizer_class.from_pretrained(tokenizer_path, do_lower_case=do_lower_case)
```

Comment:
@zphang I am concerned I am not handling the `do_lower_case` gracefully. Do you think relying on AutoTokenizers is fine in this situation?

Reply:
I wonder if it is necessary at all, actually. `.from_pretrained` ought to handle this.